justgrimes-review.js

```diff
@@ -44,10 +45,9 @@ if (Meteor.isClient) {
 }
 
 Meteor.methods({
-  addRating: function (rating) {
+  addRating: function (rating, offset) {
     if (rating > 0 && rating <= 5) {
```

Comment:
This is an example of a sanity check. No matter what you do, the server will only accept ratings between 1 and 5. (Actually, because there is no requirement that they be integers, 0.0001 is a valid rating. I should fix that.)